<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Slicer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="images_tab">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="file_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="add_files_btn">
            <property name="text">
             <string>Add files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_folder_btn">
            <property name="text">
             <string>Add Folder</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="load_project_btn">
            <property name="text">
             <string>Load project</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_project_btn">
            <property name="text">
             <string>Save project</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGraphicsView" name="preview"/>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Mask</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Blending:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="blending">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mask blending factor&lt;/span&gt;&lt;/p&gt;&lt;p&gt;0 = No-blending aka masks have soft edges&lt;/p&gt;&lt;p&gt;1 = Real soft blending&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="text">
                  <string>Forward</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QRadioButton" name="radioButton_5">
                 <property name="text">
                  <string>Backward</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Shadow alpha:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.020000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>None</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Drop shadow:</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Preview!</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Transition</string>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Transition type</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QRadioButton" name="linear_button">
                   <property name="text">
                    <string>Linear</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radial_button">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Radial</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="grid_button">
                   <property name="text">
                    <string>Grid</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="reverse_order">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reverse order of images&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Reverse order</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Linear angle</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="linear_angle">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Slice angle: &lt;/span&gt;(Linear transition)&lt;/p&gt;&lt;p&gt;0° = Horizontal slicing&lt;/p&gt;&lt;p&gt;90° = Vertical slicing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="suffix">
                  <string>°</string>
                 </property>
                 <property name="minimum">
                  <double>-90.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>90.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Radial origin:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QDoubleSpinBox" name="origin_x">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Origin&lt;/span&gt; for radial transition&lt;/p&gt;&lt;p&gt;X and Y coordinates of center of the rotation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="minimum">
                    <double>-1.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.500000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="origin_y">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="minimum">
                    <double>-1.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Scaling</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QDoubleSpinBox" name="scale_x">
                   <property name="minimum">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>6.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="scale_y">
                   <property name="minimum">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>7.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Radial coverage</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Radial start</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="radial_coverage">
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>180.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QDoubleSpinBox" name="radial_start">
                 <property name="minimum">
                  <double>-180.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>-90.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Grid rows</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QDoubleSpinBox" name="grid_rows">
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="minimum">
                  <double>2.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>9999.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>2.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Baking</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Output folder</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Output intermediate masked slices</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>JPEG</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TIFF</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Output type</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Quality</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="quality">
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>90</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="run_button">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Slice it!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="processor_info">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ready to slice!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
